### DISCUSSION TITLE
Ps2 2nd controller is connected, but input is not registered

### DISCUSSION THREAD

**Discussion by andreivlad9001** [darthcloud/BlueRetro#602] [Q&A] [answered]
I made a blue retro using  a ESP 32 devkit E with 2 directly connected controller port that is powered by microusb. It runs version 1.8.2 hw1.
I can connect 2 controllers, but the inputs of the 2nd one are not recognised

**Comment by andreivlad9001** [comment, marked as answer]
> I made a blue retro using  a ESP 32 devkit E with 2 directly connected controller port that is powered by microusb. It runs version 1.8.2 hw1.
> I can connect 2 controllers, but the inputs of the 2nd one are not recognised
> 

After rechecking countless times I've managed to make both work properly.
Although I've encountered a bug when playing need for speed titles that the controller 1's inputs don't work and the inputs of the second are recognised as the inputs of the first.

**Reply by GrishaX2** [reply]
How you make it work proparly? I have the similar problem with pre built adapter from aliexpress

**Reply by CharlieNash00** [reply]
the adapters on aliexpress are junk none of them have worked with official ps3 controllers ive tried, and I tried four controllers!

**Reply by andreivlad9001** [reply]
@GrishaX2 Check if your wires are properly connected and that there are no broken wires.
